fdbserver/Status.actor.cpp

```diff
@@ -3370,9 +3370,14 @@ ACTOR Future<StatusReply> clusterGetStatus(
 			state std::vector<JsonBuilderObject> workerStatuses = wait(getAll(futures2));
 			wait(success(primaryDCFO));
 
-			std::vector<NetworkAddress> addresses =
-			    wait(timeoutError(coordinators.ccr->getConnectionString().tryResolveHostnames(), 5.0));
-			coordinatorAddresses = std::move(addresses);
+			ErrorOr<std::vector<NetworkAddress>> addresses =
+			    wait(errorOr(timeoutError(coordinators.ccr->getConnectionString().tryResolveHostnames(), 5.0)));
+			if (addresses.present()) {
+				coordinatorAddresses = std::move(addresses.get());
+			} else {
+				messages.push_back(JsonString::makeMessage("fetch_coordinator_addresses",
+				                                           "Fetching co-ordinators addresses timed out"));
```

Comment:
Nit: typically we spell coordinator as one unhyphenated word:

```suggestion
				                                           "Fetching coordinator addresses timed out"));
```